From 41f9ed129716497a19f7ad3066b08a4ec0913325 Mon Sep 17 00:00:00 2001
From: Peter Vary <pvary@cloudera.com>
Date: Thu, 5 Jan 2017 20:35:32 +0100
Subject: [PATCH 0900/1164] CDH-43009: HIVE-14511: Improve MSCK for
 partitioned table to deal with special cases
 (Pengcheng Xiong, reviewed by Ashutosh Chauhan)  
          Updated changed q.out

Change-Id: I49ec3bdd91831d38560750d700d99b99e2468ed0
---
 .../results/clientpositive/msck_repair_0.q.out     |    7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/ql/src/test/results/clientpositive/msck_repair_0.q.out b/ql/src/test/results/clientpositive/msck_repair_0.q.out
index 3d3bef8..c394f9b 100644
--- a/ql/src/test/results/clientpositive/msck_repair_0.q.out
+++ b/ql/src/test/results/clientpositive/msck_repair_0.q.out
@@ -18,18 +18,17 @@ PREHOOK: query: MSCK TABLE default.repairtable
 PREHOOK: type: MSCK
 POSTHOOK: query: MSCK TABLE default.repairtable
 POSTHOOK: type: MSCK
-Partitions not in metastore:	repairtable:p1=c/p2=a/p3=b
+Partitions not in metastore:	repairtable:p1=c/p2=a
 PREHOOK: query: MSCK REPAIR TABLE default.repairtable
 PREHOOK: type: MSCK
 POSTHOOK: query: MSCK REPAIR TABLE default.repairtable
 POSTHOOK: type: MSCK
-Partitions not in metastore:	repairtable:p1=c/p2=a/p3=b
-Repair: Added partition to metastore default.repairtable:p1=c/p2=a/p3=b
+Partitions not in metastore:	repairtable:p1=c/p2=a
+Repair: Added partition to metastore default.repairtable:p1=c/p2=a
 PREHOOK: query: MSCK TABLE repairtable
 PREHOOK: type: MSCK
 POSTHOOK: query: MSCK TABLE repairtable
 POSTHOOK: type: MSCK
-Partitions not in metastore:	repairtable:p1=c/p2=a/p3=b
 PREHOOK: query: DROP TABLE default.repairtable
 PREHOOK: type: DROPTABLE
 PREHOOK: Input: default@repairtable
-- 
1.7.9.5

